.appveyor.yml

```diff
@@ -10,7 +11,7 @@ environment:
 
 install:
     - SET PATH=C:\\Python%PYTHON%;c:\\Python%PYTHON%\\scripts;%PATH%
-    - python -m pip install -U pip wheel setuptools
+    - pip install -U pip wheel setuptools
```

Comment:
Hmm, tricky then. https://ci.appveyor.com/project/TBBle/lxml/build/1.0.17/job/8go4rk5k1mreweyd shows that pip is up-to-date on the AppVeyor images, so I'm not hitting the issue there. I'd noticed the complaint locally, but it didn't see to fail the update as long as I updated nothing else.

I'll have a look later, there might be a bounce-script for pip that can be used with `python -m`. I've seen that in some other projects.